api/src/main/resources/liquibase.xml

```diff
@@ -5,7 +5,7 @@
     http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
 
 	<changeSet id="43889130-55c2-49e9-b72b-a9eae9a31da7" author="Partners In Health">
-	    <validCheckSum>8:0670d889112552d0e75ea01b60d8d83d</validCheckSum> <!-- old checksum before changing retire type to boolean -->
+	    <validCheckSum>3:c90c282aae153f2b5a57b61a2a697fea</validCheckSum> <!-- old checksum before changing retire type to boolean on older liquibase version -->
 	    <validCheckSum>8:373516eb011f344c988f9f5c8c121ec9</validCheckSum> <!-- new checksum after changing retire type to boolean -->
```

Comment:
I thought every checksum that starts with `8:` would be removed.

Reply:
Ok I'll add the checksums after changes, generated with older version of liquibase as well.